bin/oref0-pushover.sh

```diff
@@ -14,7 +19,61 @@ if [ -z $TOKEN ] || [ -z $USER ]; then
     exit
 fi
 
+PREF_VALUE=$(cat $PREF_FILE | jq --raw-output -r .pushover_sound 2>/dev/null)
+
+if [ ! -z $PREF_VALUE ] && [ $PREF_VALUE != "null" ]; then
+    SOUND=$PREF_VALUE
+fi
+
+PREF_VALUE=$(cat $PREF_FILE | jq .pushover_snooze 2>/dev/null)
+
+if [ ! -z $PREF_VALUE ] && [ $PREF_VALUE != "null" ] && [ "$PREF_VALUE" -eq "$PREF_VALUE" ]; then
+    SNOOZE=$PREF_VALUE
+fi
+
+PREF_VALUE=$(cat $PREF_FILE | jq --raw-output -r .pushover_only 2>/dev/null)
+
+if [ ! -z $PREF_VALUE ] && [ $PREF_VALUE != "null" ]; then
+    ONLYFOR=$PREF_VALUE
+fi
+
+PREF_VALUE=$(cat $PREF_FILE | jq .pushover_priority 2>/dev/null)
+
+if [ ! -z $PREF_VALUE ] && [ $PREF_VALUE != "null" ] && [ "$PREF_VALUE" -eq "$PREF_VALUE" ]; then
+    PRIORITY=$PREF_VALUE
+fi
+
+PREF_VALUE=$(cat $PREF_FILE | jq .pushover_retry 2>/dev/null)
+
+if [ ! -z $PREF_VALUE ] && [ $PREF_VALUE != "null" ] && [ "$PREF_VALUE" -eq "$PREF_VALUE" ]; then
+    RETRY=$PREF_VALUE
+fi
+
+PREF_VALUE=$(cat $PREF_FILE | jq .pushover_expire 2>/dev/null)
+
+if [ ! -z $PREF_VALUE ] && [ $PREF_VALUE != "null" ] && [ "$PREF_VALUE" -eq "$PREF_VALUE" ]; then
```

Comment:
Just curious, what does the last comparison in this if statement do? It seems to be comparing a variable value to itself which should always match?